eslint.config.mts

```diff
@@ -2,5 +2,12 @@ import { wrtnlabs } from "@wrtnlabs/eslint-config";
 
 export default wrtnlabs({
   typescript: false,
-  ignores: ["packages", "test", "website"],
+  ignores: [
+    // monorepo
+    "packages",
+    "test",
+    "website",
+    // pnpm store for `act` library(https://github.com/nektos/act)
+    ".pnpm-store",
```

Comment:
[imo]
I think adding `.pnpm-store` here is not a good idea because
- if you ignore `.pnpm-store`, it should be in `.gitignore`. eslint automatically ignore the files listed in `gitignore`. also we don't want to push `.pnpm-store` into repository!
- I  don't use `act` actually. If team member uses act, we should add it to `.gitignore`, but if it is only @sunrabbit123 , add it into `.config/git/ignore` and ignore in your computer.

Reply:
ok good answer
i understand it
thank you